test/helpers/createDummyCase.ts

```diff
@@ -58,8 +58,8 @@ export default async (
     triggerLockedByUsername:
       !isResolved && hasUnresolvedTriggers && randomBoolean() && triggersExist ? randomUsername() : null,
     phase: 1,
-    errorStatus: hasExceptions ? "Unresolved" : "Resolved",
-    triggerStatus: hasUnresolvedTriggers ? "Unresolved" : "Resolved",
+    errorStatus: exceptionCount === 0 ? null : hasExceptions ? "Unresolved" : "Resolved",
```

Comment:
I don't think we ever seed resolved exceptions, question is, should we?

Reply:
Good spot, I think we should.